.openshift-ci/runners.py

```diff
@@ -122,6 +126,9 @@ def log_event(self, msg, test_set=None):
         print(f"{marker} {time}: {msg}")
         print(marker)
 
+    def set_provisioned_state(self):
+        os.system("tests/e2e/lib.sh set_provisioned_state")
```

Comment:
Is the usage of `os.system` on purpose due to the [signal handling differences](https://docs.python.org/3/library/subprocess.html#replacing-os-system)?
If not, please change to `subprocess.call` or perhaps better yet, `subprocess.check_call`.